SAP/guided-development
# Context

Work in progress.
```js
// TODO:
```
* Verify `backend` tests still work
* Verify subitems still work
* Update the vscode-contrib(i) sample extensions
* Publish types to npm and reintroduce `dependencies`
